src/main/scala/org/broadinstitute/dsde/agora/server/business/AgoraBusiness.scala

```diff
@@ -102,50 +105,53 @@ class AgoraBusiness(permissionsDataSource: PermissionsDataSource)(implicit ec: E
     }
   }
 
-  private def checkValidPayload[T](agoraEntity: AgoraEntity, username: String)(op: => ReadWriteAction[T]): ReadWriteAction[T] = {
```

Comment:
Makes sense to drop the parameter, though sometimes we choose to keep 'em around to make it very obvious that sibling methods operate on similar data (I have no position on this particular one)